### PR TITLE
Fix a indent, Update latest version

### PR DESCRIPTION
**Pull request by n0ts**
Ruby 2.4.0 has been released!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The following variables may be automatically overridden with Hiera:
   "1.9.3-gentoo":
     "CC": "llvm"
     "CFLAGS": "-O9 -funroll-loops"
-  "2.0.0-p0":
+  ".0.0-p0":
```
Comment (jacobbednarz) [inline README.md]: looks like we've lost a `2` on the front of this

File: manifests/version.pp [hunk 2]
```diff
@@ -36,6 +35,8 @@
     if $::operatingsystem == 'Darwin' {
       include homebrew::config
       include boxen::config
+      ensure_resource('package', 'openssl')
```
Comment (jacobbednarz) [inline manifests/version.pp]: is openssl now a dependency of 2.4 that isn't bundled in rbenv?

**Comment by n0ts** [comment]
@jacobbednarz Thanks for your comment. I fix it. Now, rbenv(ruby-build) included openssl, ruby-build support homebrew openssl. 
ruby-build wiki , `openssl libyaml libffi package` optional recommented... I think that more fix it...
https://github.com/rbenv/ruby-build/wiki

**Comment by jacobbednarz** [comment]
released via [8.5.7](https://github.com/boxen/puppet-ruby/releases/tag/8.5.7)

**Pull request merged**